src/simularium/FrameRecorder.ts

```diff
@@ -88,18 +91,21 @@ export class FrameRecorder {
     }
 
     public async start(): Promise<void> {
-        try {
-            await this.setup();
-            this.isRecording = true;
-        } catch (e) {
-            console.log("setup failed", e);
-            return;
-        }
+        if (!this.isRecording)
+            try {
+                await this.setup();
+                this.isRecording = true;
+            } catch (e) {
+                console.log("setup failed", e);
```

Comment:
I know it's not part of this PR but the console log message needs some love.  It doesn't say WHAT setup failed.  And it makes me wonder if we need to throw a front end error or something.